src/coprocessor/xeval/evaluator.rs

```diff
@@ -82,8 +82,8 @@ impl EvalContext {
 }
 
 // `Evaluator` evaluates `tipb::Expr`.
-// TODO(performance) Evaluator should not contains any data member
-// since Managing data is not his responsibility but calculation.
+// TODO(performance) Evaluator should not contains any data member,
+// since managing data is not its responsibility but doing calculation.
```

Comment:
since managing data is not its responsibility but doing calculation. ->
because its responsibility is calculating, not managing data.